content/en/getting_started/logs/_index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Send logs to Datadog
+kind: documentation
+further_reading:
+- link: "https://learn.datadoghq.com/enrol/index.php?id=15"
+  tag: "Learning Center"
+  text: "Introduction to Logs in Datadog"
+- link: "/logs/log_collection/"
+  tag: "Documentation"
+  text: "Collect logs from your Applications, Containers, and Cloud providers"
+---
+
+## Overview
+
+Datadog Log Management collects logs from your application. This page shows you how get your first logs into Datadog. Follow the sections below:
+
+* [Create a Datadog account](#create-a-datadog-account)
```

Comment:
there's no `#create-a-datadog-account` section on this page, we can probably take this link out